features/focus-events.yml

```diff
@@ -0,0 +1,14 @@
+name: Focus events
+description: Focus events represent changes in the focused or blurred elements or windows.
+spec: https://w3c.github.io/uievents/#events-focusevent
+caniuse: focusin-focusout-events
+status:
+  compute_from: api.Element.focus_event
```

Comment:
To align with caniuse, this needs to be pinned to `focusin` and `focusout` but BCD has a very different take on whether or not to mark them supported or not. Would you mind removing all of this and filing an issue in its place, and linking the issue back to this PR discussion?

```suggestion
```

Reply:
Filed #2327